### ISSUE TITLE
Missing 4 Icons after packing with AppLovin

### ISSUE THREAD

**Issue by salvync** [distriqt/ANE-Adverts-Mediation#72]
Hi,

I am trying to verify an app (iOS), but it is giving me 4 issues: Missing icon 120x120, 167x167, 152x152 and 1024x1024 after integrating the AppLovin ane.
Everything works fine while testing. Applovin added my app bundle. 
A wild guess, could it have something to do with the AppLovinSDK.framework folder? I am using Animate.

The erros goes away when I delete all the Applovin integration requirements.

Thank you.

##### Environment 

- AIR SDK version: 
- Device OS: iOS
- Device version: 17
- Development IDE: Animate
- Development OS: MacOS

##### Logs

Device logs if relevant

**Comment by marchbold** [comment]
That sounds odd. Do you have those icons in your asset catalogue and your app descriptor?

**Comment by marchbold** [comment]
Only reason I can think of is adding the ANE could have changed the minimum sdk requirements for your app which may change the required icons ?

**Comment by salvync** [comment]
The minimum requirement on my game is 17, because it is using the Facebook API also. The facebook API was giving me errors with lower sdk versions like 13.

**Comment by salvync** [comment]
I tested with another app. No facebook API, just AppLovin (Adverts added). Minimum version 12.
It gave me the same error: 4 icons missing.

**Comment by marchbold** [comment]
Did you confirm those icons are in your asset catalogue?

**Comment by salvync** [comment]
Yes, I sent a different app to the apple store without AppLovin. It didn't give any errors. 
It only gives me the errors when I add AppLovin.
[AppLovin.txt](https://github.com/user-attachments/files/16791269/AppLovin.txt)

**Comment by marchbold** [comment]
Sure I understand that, but did you check that you have those icons in your asset catalogue? Have you generated one?
